scripts/create-api.py

```diff
@@ -3,24 +3,34 @@
 import frontmatter
 import json
 import os
+from collections import defaultdict
 
-# ... process annotations directory, just create one big
-# json blob, I think
 
 ANNOTATIONS_DIR = os.path.join(os.path.dirname(__file__), "..", "annotations")
 
+data = defaultdict(lambda: defaultdict(lambda: {}))
```

Comment:
It might be worthwhile to create a schema to validate the API:

```yaml
type: object
description: application
additionalProperties:
  type: object
  description: metric/ping
  additionalProperties:
    type: object
    properties:
      content:
        type: string
        description: markdown contents of annotation
      component:
        type: string
        $comment: could be an enumerated type
      features:
        type: string
    required:
      - content
```

Reply:
Great idea! Filed https://github.com/mozilla/glean-annotations/issues/9 for this